ledger/recovering-indexer-integration-tests/src/test/suite/scala/com/digitalasset/platform/indexer/RecoveringIndexerIntegrationSpec.scala

```diff
@@ -192,15 +192,19 @@ class RecoveringIndexerIntegrationSpec extends AsyncWordSpec with Matchers with
           restartDelay = restartDelay,
         ),
         metrics = new MetricRegistry,
-        eventsPageSize = 100,
       )(materializer, logCtx)
     } yield participantState
   }
 
   private def index(implicit logCtx: LoggingContext): ResourceOwner[LedgerDao] = {
     val jdbcUrl =
       s"jdbc:h2:mem:${getClass.getSimpleName.toLowerCase}-$testId;db_close_delay=-1;db_close_on_exit=false"
-    JdbcLedgerDao.writeOwner(ServerRole.Testing(getClass), jdbcUrl, new MetricRegistry, 100)
+    JdbcLedgerDao.writeOwner(
+      ServerRole.Testing(getClass),
+      jdbcUrl,
+      eventsPageSize = 100,
```

Comment:
Minor: I think named parameters are best used consistently in a call.

Reply:
Good point. I've added names to all other parameters.